service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/gcp/ainotebook/post-startup.sh

```diff
@@ -90,9 +90,9 @@ sudo mv nextflow /usr/bin/nextflow
 
 # Install cromwell
 readonly CROMWELL_LATEST_VERSION="81"
-sudo -u "${JUPYTER_USER}" sh -c "mkdir -p /home/${JUPYTER_USER}/cromwell"
 sudo -u "${JUPYTER_USER}" sh -c "curl -LO https://github.com/broadinstitute/cromwell/releases/download/${CROMWELL_LATEST_VERSION}/cromwell-${CROMWELL_LATEST_VERSION}.jar"
-mv "cromwell-${CROMWELL_LATEST_VERSION}.jar" "/home/${JUPYTER_USER}/cromwell/"
+sudo mv cromwell-${CROMWELL_LATEST_VERSION}.jar "/usr/share/java/"
```

Comment:
Recommend double-quoting the cromwell-<version>.jar string:

```
sudo mv "cromwell-${CROMWELL_LATEST_VERSION}.jar" "/usr/share/java/"
```

From https://google.github.io/styleguide/shellguide.html#s5.7-quoting:

> Always quote strings containing variables, command substitutions, spaces or shell meta characters, unless careful unquoted expansion is required or it’s a shell-internal integer

Arguably leaving off the quotes here is "safe" because the version variable resolves to an integer (contains no spaces), but it is a good practice.

Reply:
Thank you so much Matt. I missed that when I modify it.